nixnet/errors.py

```diff
@@ -21,8 +21,8 @@ def __init__(self, message, error_code):
         """Initialize error.
 
         Args:
-            message (string): Specifies the error message.
-            error_code (int): Specifies the NI-XNET error code.
+            message: A string specifing the error message.
+            error_code: An integer specifing the NI-XNET error code.
```

Comment:
So google's docstring style isn't very well documented.  I found some examples of it though
http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

The former is mostly correct (`str` instead of `string`)